src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onSelectedTodo: Dispatch<SetStateAction<Todo | null>>;
+  selectedUserId: number;
+  selectedTodo: Todo | null;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  onSelectedTodo,
+  selectedUserId,
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
It seems like you're initializing the user state with a null value. This is not a recommended practice as it can lead to bugs and inconsistencies in your app. Instead, you should initialize your state to the actual initial state your app should be in. In this case, you could create a default user object and use that for initialization.